wagtail/admin/templates/wagtailadmin/pages/action_menu/menu.html

```diff
@@ -3,7 +3,7 @@
     {{ default_menu_item }}
 {% endif %}
 {% if show_menu %}
-    <div class="dropdown-toggle">{% icon name="arrow-up" %}</div>
+    <div class="dropdown-toggle" aria-expanded="true" aria-controls="other-actions" tabindex="0">{% icon name="arrow-up" %}</div>
```

Comment:
I think an initial step to this would be to convert the dropdown-toggle to a `button` (remember to add `type="button"` on it also), then ensure that the styling works, and this would need to be done in all usage (about~30 in the current code).

This should solve the issue of this click target being focusable and avoids having to add JS work arounds for 'act like a button but is not really a button' issues.

That would be a recommended step 1 (focus/styles) and then the step 2 part can add the relevant aria props, currently there is nothing with an id `other-actions` that I can see so this aria-controls may not do what we want it to do.

However, let's try to solve the conversion a to an actual button first (for all usage).

Reply:
Could you let me know where else (some instances) the dropdown toggle is used (as mentioned around ~30). As the original issue focuses on the edit page, I have worked to make changes only to that page. But I would like to follow up on the others as well.